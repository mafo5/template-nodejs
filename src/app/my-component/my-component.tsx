import { ComponentProps, FunctionComponent } from "react"

export interface MyComponentProps {
    // add properties
}

export const MyComponent: FunctionComponent<MyComponentProps & ComponentProps<'div'>> = ({ className, ...props }) => {
    return <div className={className} {...props}>My Component</div>
}