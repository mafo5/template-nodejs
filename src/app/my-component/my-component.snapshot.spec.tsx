import { render } from '@testing-library/react';

import { MyComponent } from './my-component';

it('renders MyComponent unchanged', () => {
  const { container } = render(<MyComponent />);
  expect(container).toMatchSnapshot();
});
