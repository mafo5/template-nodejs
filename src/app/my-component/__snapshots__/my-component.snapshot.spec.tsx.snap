// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders MyComponent unchanged 1`] = `
<div>
  <div>
    My Component
  </div>
</div>
`;
